### ISSUE TITLE
Application Tutorial: Using Devfile to develop apps for APPUiO Cloud

### ISSUE THREAD

**Issue by tobru** [appuio/appuio-cloud-docs#87]
Create a tutorial about using [Devfile](https://devfile.io/) to write apps for APPUiO Cloud.

**Comment by akosma** [comment]
Devfile is a file format, nothing else; maybe the tutorial should be about using [odo](https://odo.dev) instead?

**Comment by tobru** [comment]
Yes, it looks like that this is tied to `odo`, I also found https://docs.openshift.com/container-platform/4.9/cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/using-devfiles-in-odo.html which shows more about this.
